h2o-algos/src/main/java/hex/tree/PlattScalingHelper.java

```diff
@@ -0,0 +1,124 @@
+package hex.tree;
+
+import hex.Model;
+import hex.ModelBuilder;
+import hex.ModelCategory;
+import hex.glm.GLM;
+import hex.glm.GLMModel;
+import water.*;
+import water.fvec.Frame;
+import water.fvec.Vec;
+
+import static hex.ModelCategory.Binomial;
+
+public class PlattScalingHelper {
+    
+    public interface ModelBuilderWithCalibration<M extends Model<M , P, O>, P extends Model.Parameters, O extends Model.Output> {
+        void warn(String field, String msg);
```

Comment:
Maybe I am blind, but I can't find where are the warn and error methods implemented...? Does the ModelBuilder override and implement these methods?

Reply:
yes these are implemented in ModelBuilder, again I thought about adding something like IModelBuilder but I really hate I-prefixed I-nterfaces so instead I went with this.

Reply:
they are already public on model builder, I know there is no pretty solution but maybe a little nicer would be to just add a method getModelBuilder() to gain access to all model builder methods? otherwise, you need to keep declaring them here whenever you need one

a similar approach with the parameter object

Reply:
done